.goreleaser.yml

```diff
@@ -222,6 +222,50 @@ nfpms:
 
 # Build container images with docker buildx (mutli arch builds).
 dockers:
+  - id: scratch-amd64
+    goos: linux
+    goarch: amd64
+    ids:
+      - collector
+    image_templates:
+      - "observiq/bindplane-agent-amd64:{{ .Major }}.{{ .Minor }}.{{ .Patch }}-minimal"
+      - "ghcr.io/observiq/bindplane-agent-amd64:{{ .Major }}.{{ .Minor }}.{{ .Patch }}-minimal"
+      - "us-central1-docker.pkg.dev/observiq-containers/agent/bindplane-agent-amd64:{{ .Major }}.{{ .Minor }}.{{ .Patch }}-minimal"
+    dockerfile: ./docker/Dockerfile.scratch
+    use: buildx
+    build_flag_templates:
+      - "--label=created={{.Date}}"
+      - "--label=title={{.ProjectName}}"
+      - "--label=revision={{.FullCommit}}"
+      - "--label=version={{.Version}}"
+      - "--platform=linux/amd64"
+    extra_files:
+      - plugins
+      - config/example.yaml
+      - config/logging.stdout.yaml
+      - LICENSE
```

Comment:
For these 'extra_files' I'd use the files that are present in the `release_deps` directory that's made by the release prep before hook. However I'm noticing that the other pre-existing docker builds don't use those files either and I'm not sure if there's a reason for that? @BinaryFissionGames Do you know?

Reply:
Yea that works for me